packages/components/src/components/radio-group/radio-group.stories.ts

```diff
@@ -41,280 +48,97 @@ export default {
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'orientation' },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
-};
-
-/**
- * Use the disabled attribute to disable a input radio. Clicks will be suppressed until the disabled state is removed
- */
-
-export const Disabled = {
-  name: 'Disabled x Size',
-  parameters: { controls: { exclude: ['size', 'default'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: [
-          {
-            type: 'slot',
-            name: 'default',
-            title: 'disabled',
-            values: [
-              {
-                value:
-                  '<sd-radio value="1" disabled>Option 1</sd-radio><sd-radio value="2">Option 2</sd-radio><sd-radio value="3">Option 3</sd-radio>',
-                title: 'true'
-              },
-              {
-                value:
-                  '<sd-radio value="1">Option 1</sd-radio><sd-radio value="2">Option 2</sd-radio><sd-radio value="3">Option 3</sd-radio>',
-                title: 'false'
-              }
-            ]
-          }
-        ],
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
-};
-
-/**
- * Use the `invalid` attribute to mark the element is not in a format or a value the application will accept.
+ * Use the `size` attribute to set the size of the radio buttons. The default is `lg`. In tight spaces, consider using `sm`.
  */
 
-export const Invalid = {
-  parameters: { controls: { exclude: ['size', 'value', 'required', 'invalid'] } },
-  render: (args: any) => {
-    return html`<form>
-      ${generateTemplate({
-        constants: [
-          { type: 'attribute', name: 'required', value: true },
-          { type: 'attribute', name: 'value', value: '' }
-        ],
-        args
-      })}
-      <sd-button style="margin-top: 16px" type="submit">Submit</sd-button>
-    </form>`;
-  },
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('sd-button');
-    await waitUntil(() => el?.shadowRoot?.querySelector('button'));
-    await userEvent.type(el!.shadowRoot!.querySelector('button')!, '{return}', { pointerEventsCheck: 0 });
-  }
+export const Size = {
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-radio-group name="radio-group" size="lg">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+        <label slot="label">Large Group</label>
+      </sd-radio-group>
+
+      <sd-radio-group name="radio-group" size="sm">
+        <sd-radio value="1">Radio 1</sd-radio>
+        <sd-radio value="2">Radio 2</sd-radio>
+        <sd-radio value="3">Radio 3</sd-radio>
+        <label slot="label">Small Group</label>
+      </sd-radio-group>
+    </div>
+  `
 };
 
```

Comment:
I would move this "required" section underneath "orientation" so it comes last.

Reply:
Updated